src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,25 @@
+.catalog {
+  display: grid;
+  place-content: center;
+  grid-template-columns: 200px;
```

Comment:
Consider using `minmax` for `grid-template-columns` to prevent the cards from stretching too much on larger screens, or being too small on smaller screens. For example, `grid-template-columns: repeat(auto-fill, minmax(200px, 1fr));` would allow the grid to adjust the number of columns based on the screen width and the minimum size of the cards.